enterprise/cypher/physical-planning/src/main/scala/org/neo4j/cypher/internal/compatibility/v3_4/runtime/SlotAllocation.scala

```diff
@@ -501,6 +508,30 @@ object SlotAllocation {
         throw new SlotAllocationFailed(s"Don't know how to handle $p")
     }
 
+  private def allocateLhsOfApply(plan: LogicalPlan,
```

Comment:
I don't see a single new unit test in slot allocation around this new code.

Reply:
We thought we could spend more time on runtime acceptance tests instead. But I added a test now.